src/applications/mhv-landing-page/utilities/data/index.js

```diff
@@ -50,129 +33,74 @@ const resolveLandingPageLinks = (
   unreadMessageAriaLabel,
   userHasHealthData = false,
 ) => {
-  // Appointments section points to VAOS on va.gov
-  const appointmentLinks = [
-    {
-      href: '/my-health/appointments/schedule/type-of-care',
-      text: 'Schedule a new appointment',
-      toggle: null,
-    },
-    {
-      href: '/health-care/schedule-view-va-appointments/appointments',
-      text: 'Upcoming VA appointments',
-      toggle: null,
-    },
-    {
-      href: '/find-locations',
-      text: 'Find VA locations',
-      toggle: null,
-    },
-  ];
-
-  const messagesLinks = resolveLinkCollection(
+  const messagesLinks = resolveLinks(
     [
       {
-        href: null,
+        ...HEALTH_TOOL_LINKS.MESSAGES[0],
         oldHref: mhvUrl(authdWithSSOe, 'secure-messaging'),
-        text: 'Inbox',
-        toggle: null,
+        oldText: 'Inbox',
```

Comment:
Why not just have a different set of objects chosen by the feature flag? May be a bit more flexible.

Reply:
Could you demonstrate what you mean?

Reply:
```js
if (featureToggles[FEATURE_FLAG_NAMES.mhvLandingPageEnableVaGovHealthToolsLinks]) {
  medicationsLinks = [
      {
        href: mhvUrl(authdWithSSOe, '/prescription-tracking'),
        text: 'Track prescription delivery'
      },
    ];
   messagesLinks = ...
} else {
   medicationsLinks = [
    {
        href: null,
        text: 'Track prescription delivery',
      },
  ];
  messagesLinks = ...
};
```
Something like that. The if block can have all the new links in the one block. I am just thinking that could be easier to read what belongs to a specific toggle.

Reply:
I think we can get to a cleaner/clearer implementation, eventually. For now this is an iterative improvement over what we have.